describealign.py

```diff
@@ -1,3 +1,19 @@
+# nuitka-project-if: {OS} != "Windows":

```

Comment:
Not a big fan of this being above the "what does the script do" header, can it be moved below that?

Reply:
Yep, should work

Reply:
Done